### DISCUSSION TITLE
How to display titles as a list?

### DISCUSSION THREAD

**Discussion by mozonte** [datenstrom/community#443] [Ask a question] [answered]
Hi, Ive been using (and liking) your CMS for a few months now. 

Id like to print the (linked) titles  on the front page. What file is the main page? and what php should i insert there?

Also, is there something that will print just titles from a given tag?

(Im using 'Blog')

Thanks

**Comment by hysonlee** [comment]
You can use shortcuts to show titles about your blog, e.g. New `[blog changes /]` ,Showing list of tags `[blogtags / 10]`. Use template (.html) and API you can customize your ouputing contents.

**Comment by mozonte** [comment]
Im not sure what you mean?

I tried putting those on the front page (front end and page.md) but that cause the page to do: 

Server error
Something went wrong. Blogtags '/' does not exist!

Basically what Im trying to do is similar to the naviagation, where it prints the titles and articles for a given number of posts, paginated.

But I want to print just the titles (separated by commas or as a list or whatever) which link to the articles, so I can have access to all the blog posts from the front page.

It would be better if I could specify this, so it would print all the blog posts that have the #history tag, then under that, all the blog posts that have the #pandas tag (I know how to do this in WordPress, but not sure how with Datenstrom)

**Comment by hysonlee** [comment]
Sorry that can't help you, suggest reading the documentations for 'blog' and 'shutcuts' further.
You mentioned you're using Blog, so I assume your blog is located on the home page,if not ,try to modify the shortcut path `[blogchanges /blog/]`.

**Comment by mozonte** [comment]
You're right. [blogchanges /blog/] printed something. it printed the 5 most recent blog titles!

maybe i just dont undestand what you mean with the first 2 examples you gave. is there a list somewhere of these shortcuts?

i wasnt able to find a way to modify [blogchanges /blog/] to make it do other stuff. do you have any idea how i could? like modify it so it prints only blogs with a certain tag, or change the number of blog titles it prints (all), or format it so i can comma separate the titles (instead of a list)?

EDIT: one part I understand now. You can have unlimited titles printed, while keeping a limited (to 5 in my case) items in the pagination this way:

BlogPagesMax: 0
BlogPaginationLimit: 5

i also formatted the printout using css with this:

li::after {
  content: ", ";
}
li {display:inline;}

However, I don't understand the php and still am stumped on how to print out only stuff from a certain tag.

**Comment by mozonte** [comment]
bump?

**Comment by schulle4u** [comment, marked as answer]
As already suggested, you should consider reading the appropriate extension documentation. The shortcuts for displaying blog information for example are very flexible, and in most cases you don't need to touch the php code. 

To display latest blog post for a specific tag: `[blogchanges /blog/ 10 - exampletag]`
To display blog pages alphabetically for a specific tag: `[blogpages /blog/ 0 - exampletag]`

Simply put the desired shortcut into your markdown pages, and Yellow will do the rest. 

See also: https://github.com/datenstrom/yellow-extensions/blob/master/features/blog/README.md

If you need more flexibility, feel free to give a more detailed explanation of what you would like to achieve. 

Steffen

**Comment by mozonte** [comment]
Awesome, @schulle4u 

That totally works. Thanks also for the link. I didn't know about that documentation before, but yeah it looks like youre right there's shortcuts for tons of things.

#resolved

**Comment by annaesvensson** [comment]
Feel free to close your own issues when they are resolved.
